pkg/gui.py

```
@@ -10,16 +10,27 @@
 import psutil
+import pytesseract
+import tkinter as Tk
 from pubsub import pub
+from tkinter import Button, Label, messagebox, OptionMenu, StringVar, TclError, Toplevel, N, S, E, W
+
 from .TIPA import ProcessManager
+from logger_config import logger
+
+
+if os.path.exists("settings.json"):
+    with open("settings.json", "r") as settings_file:
```

Ruff (0.8.0)

Unnecessary open mode parameters

Remove open mode parameters

(UP015)